SDK/AppCenter/Microsoft.AppCenter.Windows.Shared/Utils/UserIdEventArgs.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+
+namespace Microsoft.AppCenter.Windows.Shared.Utils
+{
+    /// <summary>
+    /// Event args for event that occurs when an user change is received.
+    /// </summary>
+    public class UserIdUpdatedEventArgs : EventArgs
+    {
+        /// <summary>
+        /// User Id
+        /// </summary>
+        public string userId;
```

Comment:
Use a property (and internal set;)

Reply:
Also capitalize the name - should be UserId

Reply:
@guperrot Setting it as internal does not allow me access from Push.cs, can I make this a public property?